base/src/main/java/com/smartdevicelink/managers/lifecycle/BaseLifecycleManager.java

```diff
@@ -406,6 +413,7 @@ public void onReceived(RPCMessage message) {
                                     msg.setCorrelationID(UNREGISTER_APP_INTERFACE_CORRELATION_ID);
                                     sendRPCMessagePrivate(msg, true);
                                     clean();
+                                    lifecycleListener.onError(null, TransportConstants.UNSUPPORTED_VEHICLE_INFO_REASON, null);
```

Comment:
Could you remove this as onClose gets called right after which will lead to onDestroy in `SdlService`.
```suggestion
```

Reply:
Removed (23ad06b)